shipyard_airflow/control/tasks.py

```diff
@@ -0,0 +1,43 @@
+# Copyright 2017 AT&T Intellectual Property.  All other rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import falcon
+import json
+import requests
+
+from .base import BaseResource
+
+class TaskResource(BaseResource):
+
+    authorized_roles = ['user']
+
+    def on_get(self, req, resp, dag_id, task_id):
+        # Retrieve URL
+        web_server_url = self.retrieve_config('BASE', 'WEB_SERVER')
+
+        if 'Error' in web_server_url:
+            resp.status = falcon.HTTP_400
+            resp.body = json.dumps({'Error': 'Missing Configuration File'})
+            return
+        else:
+            req_url = '{}/api/experimental/dags/{}/tasks/{}'.format(web_server_url, dag_id, task_id)
+            task_details = requests.get(req_url).json()
+
+            if 'error' in task_details:
```

Comment:
Actually the rationale for doing this is due to the airflow api codes itself.  It already has error handling codes and the output will include 'error_message' in it, i.e. https://github.com/apache/incubator-airflow/blob/master/airflow/api/common/experimental/get_task.py.  


For instance, I did the following tests and were able to get the expected result:

ubuntu@ubuntu:~$ http localhost:9000/api/experimental/dags/test_dag/tasks/test1 X-Auth-Token:admin
HTTP/1.1 400 Bad Request
content-length: 49
content-type: application/json; charset=UTF-8
x-shipyard-req: d713b49a-94e6-4f9e-8337-082dd6c86f1d

{
    "error": "Task test1 not found in dag test_dag"
}

ubuntu@ubuntu:~$ http localhost:9000/api/experimental/dags/test1/tasks/sleep X-Auth-Token:admin
HTTP/1.1 400 Bad Request
content-length: 32
content-type: application/json; charset=UTF-8
x-shipyard-req: bad1cf05-b41f-41e8-877e-74a6d5543f89

{
    "error": "Dag test1 not found"
}